tsconfig.json

```diff
@@ -14,6 +11,7 @@
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
     "resolveJsonModule": true,
-    "allowJs": false
+    "allowJs": false,
+    "types": ["jest", "jest-extended"]
```

Comment:
There should be no need for this, as all other types will be excluded.

Have you checked this maybe? https://jest-extended.jestcommunity.dev/docs/getting-started/typescript

Reply:
> There should be no need for this, as all other types will be excluded.
> 
> Have you checked this maybe? https://jest-extended.jestcommunity.dev/docs/getting-started/typescript

With that approach `preconstruct` fails https://github.com/commercetools/test-data/actions/runs/10812738929/job/29995067358?pr=681#step:7:11

Reply:
> > There should be no need for this, as all other types will be excluded.
> > Have you checked this maybe? https://jest-extended.jestcommunity.dev/docs/getting-started/typescript
> 
> With that approach `preconstruct` fails https://github.com/commercetools/test-data/actions/runs/10812738929/job/29995067358?pr=681#step:7:11

I replaced `"files": ["global.d.ts"]` with `"typeRoots": ["./node_modules/@types", "./global.d.ts"]` and it worked. It doesn't exclude any type present in  `./node_modules/@types` and it adds the needed `jest-extended` types.